provider/flexibleengine/flexibleengine.go

```diff
@@ -101,7 +101,7 @@ func (a Provider) ResourceInOut(id, rs string, cfgs map[string]map[string]interf
 			ins = append(ins, pid.(string))
 		}
 	}
-	return ins, outs
+	return ins, outs, nil
```

Comment:
The change of prototype/return is just for AWS isn't it?

Reply:
Yes for now.

Reply:
I did also update the README as I forgot :)